index/upsidedown/reader.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/blevesearch/bleve/v2/size"
 	index "github.com/blevesearch/bleve_index_api"
-	"github.com/blevesearch/upsidedown_store_api"
+	store "github.com/blevesearch/upsidedown_store_api"
```

Comment:
Unrelated change?

Reply:
This popped up because of the vscode extension I believe, removing it.